index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Change required:
A proper page title is suggested. It's better to bring to the user the general idea of the web page, and its functionality.

Reply:
- [ ] Also impressive work you did here 👏 , but please refer to my detailed review on line `18` on highlighting tags and use of emojis 👍